apps/guardian-ui/src/components/GuardiansCard.tsx

```diff
@@ -82,6 +82,10 @@ export const GuardiansCard: React.FC<Props> = ({ status, config }) => {
     return peerDataArray;
   }, [status, config, t]);
 
+  if (!Object.keys(status?.federation?.status_by_peer || {}).length) {
+    return null;
+  }
```

Comment:
Any reason this isn't just `rows.length`? If the intent is to render the loading state, maybe `config && rows.length`?

Reply:
fixed, just checking if it's solo, using rows makes more sense